.github/workflows/test-helm-chart.yaml

```diff
@@ -48,10 +43,9 @@ jobs:
       # ref: https://github.com/jupyterhub/action-k3s-helm/
       - uses: jupyterhub/action-k3s-helm@v1
         with:
-          k3s-channel: v1.19
```

Comment:
this is probably the most important fix of this PR

Reply:
:facepalm: